apps/storefront/app/bloggen/2025/helptext-blir-fjerna-kva-gjer-du/page.mdx

```diff
@@ -0,0 +1,161 @@
+import { PostLayout, Contributors } from '../../_components';
+import { ResponsiveIframe } from '@components';
+import { CodeSnippet } from '@repo/components';
+
+export default ({ children }) => (
+  <PostLayout
+    content={children}
+    heading='HelpText blir fjerna, kva gjer du?'
+    ingress='Me fjernar HelpText frå Designsystemet. Kvifor, og kva gjer du om du fortsatt vil bruke den?'
+    date='7. januar 2025'
+    author='Designsystemet'
+    imageSrc='/img/bloggen/helptext.png'
+    imageAlt='Illustrasjon av ein hjelpetekst i popover.'
+    imageCaption='HelpText er i dag ein popover knytt til ein knapp.'
+  />
+);
+
+export const metadata = {
+  title: 'HelpText blir fjerna, kva gjer du?',
+  description:
+    'Me fjernar HelpText frå Designsystemet. Kvifor, og kva gjer du om du fortsatt vil bruke den?',
+  openGraph: {
+    images: '/img/bloggen/helptext.png',
+  },
+};
+
+I `next`-versjonen av Designsystemet skal me fjerne `HelpText`, dette betyr at den ikkje blir med i versjon 1.
```

Comment:
Suggested wording to make it more readable in past tense, since people will most likely come here after its removed? 
```suggestion
I `1.0.0.next`-versjonen av Designsystemet fjernet me `HelpText`, dette betyr at den ikkje blir med i versjon 1.
```

Reply:
I have committed a suggestion